# Remove read-only-files test.
# Fail to build on btrfs with compression enabled.
#
# [44] patch -p0 --read-only=fail < f.diff || echo "Status: $?" -- ok
# [54] patch -f -p0 --read-only=warn < f.diff || echo "Status: $?" -- FAILED
# --- expected
# +++ got
# @@ -1,4 +1,4 @@
#  File f is read-only; trying to patch anyway
#  patching file f
# -File f is read-only; trying to patch anyway
# -patching file f
# +/home/seblu/arch/packages/core/patch/trunk/src/patch-2.7.6/src/patch: setting attribute btrfs.compression for btrfs.compression: Permission denied
# +Status: 2
# [65] patch -f -p0 --read-only=ignore < f.diff || echo "Status: $?" -- FAILED
# --- expected
# +++ got
# @@ -1,2 +1,3 @@
#  patching file f
# -patching file f
# +/home/seblu/arch/packages/core/patch/trunk/src/patch-2.7.6/src/patch: setting attribute btrfs.compression for btrfs.compression: Permission denied
# +Status: 2
# 3 tests (1 passed, 2 failed)
# FAIL read-only-files (exit status: 1)


--- a/tests/Makefile.am	2023-02-12 21:53:41.703257940 +0100
+++ b/tests/Makefile.am	2023-02-12 22:01:28.252357777 +0100
@@ -55,7 +55,6 @@
 	preserve-c-function-names \
 	preserve-mode-and-timestamp \
 	quoted-filenames \
-	read-only-files \
 	reject-format \
 	remember-backup-files \
 	remember-reject-files \
